packages/grant-explorer/src/features/api/rounds.ts

```diff
@@ -96,32 +96,13 @@ async function fetchRoundsByTimestamp(
   }
 }
 
-/**
- * Filters out testnet chainIds in production, otherwise returns all active chainIds
- * */
-const getActiveChainIds = () =>
-  Object.keys(ChainId).filter((chainId) => {
-    const chainDefinition = allChains.find(
-      (chain) => chain.id === Number(chainId),
-    );
-
-    /* This part could be made shorter, but is intentionally verbose to ease
-     * underestanding */
-    if (
-      process.env.REACT_APP_ENV === "production" &&
-      chainDefinition?.testnet
-    ) {
-      return false;
-    }
-
-    return true;
-  });
+const activeChainIds = allChains
```

Comment:
I actually liked that you had a function here. If you have two pieces of state, they can get out of sync. If you have one piece of state and one function that derives a value from it, pay a tiny computational price for the reassurance that the value you get is up to date. Not a bad deal.